pom.xml

```diff
@@ -1308,6 +1310,37 @@
     </build>
 
     <profiles>
+        <profile>
+            <!-- Adds the `documentation-generation` module as an annotation
+            processor to scan the classpath and extract bug pattern data used
+            in documentation generation. Assumes that a prior build has already
+            installed the project in the local Maven repository. -->
```

Comment:
I recall that we discussed offline at some point that we could instead use the same approach as we do for `refaster-compiler`. Was that angle assessed further? :)

Reply:
Yes, added a commit for this! Thanks for reminding me 😉.